app/javascript/application.js

```diff
@@ -0,0 +1,9 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+import "jquery"
+import "bootstrap"
+
+// Add jQuery to global scope for legacy code
+window.jQuery = jQuery;
+window.$ = jQuery;
+
+console.log('Hello World from Importmap')
```

Comment:
This feels unneeded, but also not blocking.

Reply:
It was quite a nice wa to ensure importmaps was working since I ran into weirdness with another app.